### PR TITLE
Update AlignedChatBox.css for 6/28/24

### PR DESCRIPTION
**Pull request by Atvaster**
I like your code mrs code

### CODE REVIEW

File: discord/AlignedChatBox.css [hunk 1]
```diff
@@ -1,10 +1,10 @@
 /* ----- Aligned Chatbox -----
   Squares up chatbox with user control area */
 :is(main,section)[class^=chatContent] > form,
-form > div[class^=submitContainer] {
+form > div {
```
Comment (CodeF53) [inline discord/AlignedChatBox.css]: No. You can't just apply styles to every div with a form as it's parent.

**Review by CodeF53 — CHANGES_REQUESTED** [review]

File: discord/AlignedChatBox.css (hunk 1 above)
Comment (Atvaster) [reply discord/AlignedChatBox.css]: I made it better

**Comment by CodeF53** [comment]
Will properly review June 30th. Can't test till then

File: discord/AlignedChatBox.css [hunk 2]
```diff
@@ -29,19 +28,8 @@ form > div[class^=submitContainer] {
...
-  > div > div > div[class^=attachedBars] > div {
+  > div > div > div[class^=attachedBars] {
     min-height: unset !important;
     padding-bottom: 4px;
   }
 }
-
-/* jump to present */
-div[class^=jumpToPresentBar] {
-  left: 0; right: unset;
```
Comment (CodeF53) [inline discord/AlignedChatBox.css]: oops

**Review by CodeF53 — APPROVED** [review]

**Pull request merged**